Add scored fire traits from floras/taxonomic literature: Yang_2023

yangsophieee opened the pull request.

ehwenk commented on data/Mitchell_2008/data.csv:
will remove - my mistake

ehwenk commented on data/NTH_2023/data.csv:
also needsto be removed - I'm doing it

ehwenk reviewed (approved):
This looks excellent - especially once I fixed the mistakes I introduced by merging all the datasets together.

I've also double checked excluded data, and summarised by taxon & trait to look at the summary patterns

dfalster reviewed (changes requested):
This is an amazing amount of detailed work, well done!

Please make the following minor changes to `Yang_2023/data.csv` 

- the file contains some special characters that get funky when opened in excel. E.g. … , – (long hyphen). Can you please find a replace (use VScode or other text editor, not Excel)
- can we reorder the columns ? E.g. put source_ID, year. Then maybe the columns with scraped text and references, all before the extracted /scored values
- replace NA with "", as it makes the spreadsheet much easier to review, either in text editor or Excel. 

I can't give any meaningful feedback on the actual traits and definitions, Lizzy tells me you have discussed these in detail and there is also a workshop coming where these will be reviewed. So that's the best place for feedback on these.

yangsophieee commented:
@dfalster Thank you for the review! I forgot to turn email notifications back on so I'm very late to fixing it, sorry about that. I merged develop into this branch to fix some conflicts. Let me know if everything is all good now and I'll squash and merge :)

yangsophieee commented:
Added more polishing of data file, including merging, renaming and removing columns. See new column names, including `excerpt_with_fire`, `fire_response_scoring_method` and so on. Clarified phrasing in the `..._scoring_method` columns by changing `direct_from_flora` to `scored_from_excerpt`. Removed two new traits with very little records: `plant_tolerance_fire` and `plant_influence_on_fire_risk`. 

Thank you @dfalster for the great suggestions! Let me know when it's ready to merge in.

dfalster commented:
Thanks for revising @yangsophieee , much improved!
